wandb/docker/__init__.py

```diff
@@ -114,6 +116,12 @@ def build(
     return stdout
 
 
+def should_add_load_argument(platform: Optional[str]) -> bool:
```

Comment:
Can you leave a comment here?  Failing to understand why the platform parameter determines the need for this

Reply:
Added